src/Forms/Forms.js

```diff
@@ -98,7 +107,7 @@ export const FormMessage = ({ type, children, className, ...props }) => {
 FormMessage.propTypes = {
     children: PropTypes.node,
     className: PropTypes.string,
-    type: PropTypes.oneOf(['', 'error', 'warning', 'help'])
+    type: PropTypes.oneOf(['error', 'warning', 'help'])
```

Comment:
@mobot11 : looks like this was missed in the earlier const PR

Reply:
We should either fix this or create a follow-up issue.